corehq/motech/repeaters/views/tests/test_repeat_records.py

```diff
@@ -76,60 +76,3 @@ def test__change_record_state(self):
             self.assertEqual(result, expected_result)
-
-    @patch('corehq.motech.repeaters.views.repeat_records.task_generate_ids_and_operate_on_payloads')
-    @patch('corehq.motech.repeaters.views.repeat_records.expose_cached_download')
-    def test__schedule_task_with_flag_no_query(self,
-                                               mock_expose_cache_download,
-                                               mock_task_generate_ids_and_operate_on_payloads):
-        mock_request = Mock()
-        query_dict = QueryDict('a=1&b=2')
-        mock_request.POST = query_dict
-        mock_domain = 'domain_1'
-        mock_action = 'action_1'
-
-        repeat_records._schedule_task_with_flag(mock_request, mock_domain, mock_action)
-        self._mock_schedule_task(query_dict, mock_domain, mock_action,
-                                 mock_expose_cache_download,
-                                 mock_task_generate_ids_and_operate_on_payloads)
-
-    @patch('corehq.motech.repeaters.views.repeat_records.task_generate_ids_and_operate_on_payloads')
-    @patch('corehq.motech.repeaters.views.repeat_records.expose_cached_download')
-    def test__schedule_task_with_flag_with_query(self,
-                                                 mock_expose_cache_download,
-                                                 mock_task_generate_ids_and_operate_on_payloads):
-        mock_request = Mock()
-        query_dict = QueryDict('a=1&b=2')
-        mock_request.POST = query_dict
-        domain = 'domain_1'
-        action = 'action_1'
-
-        repeat_records._schedule_task_with_flag(mock_request, domain, action)
-        self._mock_schedule_task(query_dict, domain, action,
-                                 mock_expose_cache_download,
-                                 mock_task_generate_ids_and_operate_on_payloads)
-
-    @patch('corehq.motech.repeaters.views.repeat_records.task_operate_on_payloads')
-    @patch('corehq.motech.repeaters.views.repeat_records.expose_cached_download')
-    @patch('corehq.motech.repeaters.views.repeat_records._get_record_ids_from_request')
-    def test__schedule_task_without_flag(self, mock__get_records, mock_expose_cache_download,
-                                         mock_task_operate_on_payloads):
-        mock_request = Mock()
-        record_id_values = ['', None, 'a=1&b=2']
-        mock_request.POST.get.side_effect = record_id_values
-        domain = 'domain_1'
-        action = 'action_1'
-
-        for __ in record_id_values:
-            repeat_records._schedule_task_without_flag(mock_request, domain, action)
-            mock__get_records.assert_called_with(mock_request)
-            mock_records_ids = mock__get_records(mock_request)
-            self._mock_schedule_task(mock_records_ids, domain, action,
-                                     mock_expose_cache_download, mock_task_operate_on_payloads)
-
-    def _mock_schedule_task(self, data, domain, action, expose_cache_download, task_to_perform):
```

Comment:
This tests the parameters that are passed to tasks ... which seems like a lot of fragile code without a lot of benefit. This function also assumes that two different tasks that are called have the same number of parameters, which is no longer true. So I decided just to delete these tests.